src/App.tsx

```diff
@@ -1,7 +1,29 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
not so good approach for naming, it can be two ways: if you left as it is, you should name variable, smth like "pressedKeyMessage", or here you should save only pressed key